tool/tctl/common/collection.go

```diff
@@ -524,7 +526,8 @@ func (c *appCollection) writeText(w io.Writer) error {
 	for _, app := range c.apps {
 		labels := stripInternalTeleportLabels(c.verbose, app.GetAllLabels())
 		rows = append(rows, []string{
-			app.GetName(), app.GetDescription(), app.GetURI(), app.GetPublicAddr(), labels, app.GetVersion()})
+			app.GetName(), app.GetDescription(), app.GetURI(), app.GetPublicAddr(), labels, app.GetVersion(),
```

Comment:
Not a huge deal, but I don't see a great reason to include these formatting changes here and shift the git blame to you.